variants/particle_xenon/variant.h

```diff
@@ -154,13 +148,13 @@ static const uint8_t A5  = PIN_A5 ;
  */
 #define SPI_INTERFACES_COUNT 2
 
-#define PIN_SPI_SS           PIN_A0
-#define PIN_SPI_MISO         _PINNUM(1, 14)
-#define PIN_SPI_MOSI         _PINNUM(1, 13)
-#define PIN_SPI_SCK          _PINNUM(1, 15)
-#define PIN_SPI1_MISO        _PINNUM(1, 8)
-#define PIN_SPI1_MOSI        _PINNUM(1, 2)
-#define PIN_SPI1_SCK         _PINNUM(1, 1)
+#define PIN_SPI_SS                  PIN_A0
+#define PIN_SPI_MISO                _PINNUM(1, 14)
```

Comment:
all PIN_ in variant is mapped through `g_ADigitalPinMap`, you shouldn't use _PINNUM() for these peripherals, please double check it, it should use the Dx, or simply and index to the mapping array. 
https://github.com/adafruit/Adafruit_nRF52_Arduino/blob/master/libraries/SPI/SPI.cpp#L67

Check out feather nrf52840 for example, 52832 is only version we do direct mapping https://github.com/adafruit/Adafruit_nRF52_Arduino/blob/master/variants/feather_nrf52840_express/variant.h

Reply:
Fixed!